File: 03.business-os/PocketSecretary/iphone/PocketSecretary/BusinessOSCommonOSLaunch/BusinessOSCommonOSLaunchRegistry.swift
import Foundation

enum BusinessOSCommonOSLaunchRegistry {
    static func createDefault() -> BusinessOSCommonOSLaunchDescriptor {
        BusinessOSCommonOSLaunchDescriptor(
            appName: "PocketSecretary",
            launchMode: "launch_via_commonos_consumer",
            defaultTarget: "businessos-commonos-consumer",
            consumerTarget: "_commonos",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            uiOwner: "CommonOS",
            businessOwner: "PocketSecretary"
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/launch/BusinessOSCommonOSLaunchRegistry.java
package com.lsam.PocketSecretary.launch;

public final class BusinessOSCommonOSLaunchRegistry {

    public BusinessOSCommonOSLaunchDescriptor createDefault() {
        return new BusinessOSCommonOSLaunchDescriptor(
                "PocketSecretary",
                "launch_via_commonos_consumer",
                "businessos-commonos-consumer",
                "_commonos",
                "shared_provider",
                "os_side_consumer",
                "CommonOS",
                "PocketSecretary"
        );
    }
}
